test/server/cards/02_SHD/units/MandalorianWarrior.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('Mandalorian Warrior', function () {
+    integration(function () {
+        describe('Mandalorian Warrior\'s ability', function () {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['mandalorian-warrior'],
+                        groundArena: ['protector-of-the-throne', 'battlefield-marine'],
+                    },
+                    player2: {
+                        groundArena: ['snowspeeder']
+                    }
+                });
+            });
+
+            it('should give an experience to mandalorian unit', function () {
+                // add experience to a mandalorian
```

Comment:
Don't necessarily need to add comments for really simple tests like this, where the test name pretty much explains the whole test. Not necessarily a problem, if you find it helpful then definitely keep doing it. Just not a requirement unless the test is more complex